styleguide/README.md

```diff
@@ -91,3 +91,9 @@ To install a near-empty StarterKit as a starting point for your project type:
 To install a specific StarterKit from GitHub type:
 
     php core/console --starterkit --install <starterkit-vendor/starterkit-name>
+
+
+### Slim and remove bloat from SVG files: install and run SVGO
```

Comment:
Sorry to keep pushing back, but this should go in the root readme, not the PL-specific one. Also, please add some narrative as I described above. You should work under the assumption that the user reading the documentation doesn't know what SVGO is or when/why they should use it.